pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -221,7 +230,7 @@ func (configgen *ConfigGeneratorImpl) buildSidecarListeners(env *model.Environme
 		// add an extra listener that binds to the port that is the recipient of the iptables redirect
 		listeners = append(listeners, &xdsapi.Listener{
 			Name:           VirtualListenerName,
-			Address:        util.BuildAddress(WildcardAddress, uint32(mesh.ProxyListenPort)),
+			Address:        util.BuildAddress(actualWildcard, uint32(mesh.ProxyListenPort)),
```

Comment:
Will it work if both IP4 and IP6 are available ? AFAIK it should, but not sure how to test.